packages/elements/package.json

```diff
@@ -41,6 +41,7 @@
     "@types/react": ">=17.0.5",
     "@types/react-dom": ">=17.0.3",
     "@types/styled-system": ">=5.1.10",
+    "csstype": ">=3.0.8",
```

Comment:
This dep can be removed, see comment below.

Reply:
fixed